backend/selectgen.ml

```diff
@@ -351,6 +351,7 @@ module Effect_and_coeffect : sig
 
   val effect_only : Effect.t -> t
   val coeffect_only : Coeffect.t -> t
+  val effect_and_coeffect : Effect.t -> Coeffect.t -> t
```

Comment:
(The naming feels confusing: `effect_only` and `coeffect_only`
are projections, but `effect_and_coeffect` is not.)

Reply:
All three are constructors for `t`.

Reply:
Indeed... I was confused.